microsetta_public_api/api/microsetta_public_api.yml

```diff
@@ -660,9 +660,43 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  type: string
+                $ref: '#/components/schemas/datasetDetail'
+
+  '/dataset/{dataset}':
+    parameters:
+      - $ref: '#/components/parameters/namedDataset'
+    get:
+      operationId: microsetta_public_api.api.datasets.dataset_detail
+      summary: Get detail on a particular dataset
+      description: Get detail on a particular daaset
```

Comment:
I realize the specifics of the what is returned are given below in the responses schema, but for some reason `detail` also seems a bit vague. Able to provide a little more text about the type of information a 'detail' is supposed to encapsulate